scene/2d/remote_transform_2d.cpp

```diff
@@ -102,6 +116,41 @@ NodePath RemoteTransform2D::get_remote_node() const {
 	return remote_node;
 }
 
+void RemoteTransform2D::set_use_global_coordinates(const bool p_enable) {
+	use_global_coordinates = p_enable;
+}
```

Comment:
This was intentional.

I wasn't sure if I should update the remote based on changes in coordinates, since this can lead to the remote's transform being changed by accident (say someone accidentlly toggles the wrong option) and since it edits the node's transform, it could be a hassle to move back to it's original position. I don't think Ctrl-Z could undo the transform change. I'll test, and if it does I'll make the change for sure

I'm not against changing it though, but I thought I'd explain the reasoning behind why I left it out. I'll change it if you think it's not going to be a problem or undo works.